.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,98 @@
+name: "Continuous Integration"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - 'refs/pull/*'
+
+env:
+  TEST_TAG: laminas/ci-matrix-action:test
+
+jobs:
+  matrix:
+    name: Generate job matrix
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.matrix.outputs.matrix }}
+    steps:
+      - name: Checkout sourcecode
+        uses: actions/checkout@v3
+      - name: Gather integration test directory names
+        id: matrix
+        run: cd tests; echo "::set-output name=matrix::[\"$(ls -d * | tr '\n' ' ' | sed 's/ $//' | sed 's/ /\",\"/g')\"]"
```

Comment:
Maybe a bit cryptic - TBH, a few comments in this file would clarify that we're running a bunch of parallel tests to check that the action will generate the right JSON for the various directories in here.

That said, this is 💯 

Awesome!

Reply:
True. Will add this as a comment. Will merge this afterwards and then rebase #83